src/call_plots.jl

```diff
@@ -576,36 +576,72 @@ function stair_plot(results::Array, variables::Array; kwargs...)
     return stair_plot(new_results; kwargs...)
 end
 ################################### DEMAND #################################
+
+function _shorten_time(results::IS.Results, horizon::Int64, init_time::Dates.DateTime)
+    time_range = IS.get_time_stamp(results)[:, 1]
+    resolution = Dates.Hour(time_range[2] - time_range[1])
+    parameters = IS.get_parameters(results)
+    times = [init_time:resolution:(init_time + resolution * horizon)]
+    new_time_range = DataFrames.DataFrame(:Range => vcat(times...))
+    new_params = Dict()
+    init_spot = findall(x -> x == init_time, time_range)[1]
+    if isempty(init_spot)
```

Comment:
I think `init_time in time_range` should work

Reply:
I did init_spot to grab the specific location, for example 2 pm might be the 7th time in the array, so the load needs to be from the 7th spot...